tracer/build/_build/Build.Steps.cs

```diff
@@ -1454,30 +1504,21 @@ static LogLevel ParseNativeLogLevel(string value)
 
     private void EnsureResultsDirectory(Project proj) => EnsureCleanDirectory(GetResultsDirectory(proj));
 
-    private (string, string) GetUnixArchitectureAndExtension(bool includeMuslSuffixOnAlpine)
-    {
-        return (IsOsx, IsAlpine, includeMuslSuffixOnAlpine) switch
-        {
-            (true, _, _) => ("osx-x64", "dylib"),
-            (_, true, true) => ($"linux-musl-{LinuxArchitectureIdentifier}", "so"),
-            _ => ($"linux-{LinuxArchitectureIdentifier}", "so"),
-        };
-    }
-
+    private (string, string) GetUnixArchitectureAndExtension() => IsOsx ? ("osx-x64", "dylib") : ($"linux-{LinuxArchitectureIdentifier}", "so");
     // the integration tests need their own copy of the profiler, this achived through build.props on Windows, but doesn't seem to work under Linux
```

Comment:
not part of your changes, but while we're in here...
```suggestion
    // the integration tests need their own copy of the profiler, this achieved through build.props on Windows, but doesn't seem to work under Linux
```